gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -668,3 +593,45 @@ def add_arguments(self, parser):
               'be the same as the BigQuery table, but it requires all '
               'extracted variants to have the same call name ordering (usually '
               'true for tables from single VCF file import).'))
+
+
+def _validate_inputs(parsed_args):
+  if ((parsed_args.input_pattern and parsed_args.input_file) or
+      (not parsed_args.input_pattern and not parsed_args.input_file)):
+    raise ValueError('Exactly one of input_pattern and input_file has to be '
+                     'provided.')
+  if parsed_args.input_pattern is not None:
```

Comment:
Can be simplified as `if parsed_args.input_pattern:`

Reply:
Since we also have similar validations in `_get_input_patterns` and `_get_file_names` (we have to validate the input pattern listed in the `input_file` as well), I prefer to move all validations except the first statement to `_get_file_names`. And the code can be simplified:
1. If `input_file` is provided, check whether it exists and read the contents and assign to known_args.input_pattern.
2. Otherwise, convert known_args.input_pattern to [known_args.input_patterns].
3. Now we can validate every item in input_pattern.

Reply:
Done.